modules/flowable-form-spring/src/test/java/org/flowable/spring/test/autodeployment/SpringAutoDeployTest.java

```diff
@@ -134,34 +132,34 @@ public void testBasicFlowableSpringIntegration() {
         expectedFormDefinitionKeys.add("form1");
         expectedFormDefinitionKeys.add("form2");
 
-        assertEquals(expectedFormDefinitionKeys, formDefinitionKeys);
+        assertThat(formDefinitionKeys).isEqualTo(expectedFormDefinitionKeys);
```

Comment:
We can get rid of the loop and expected form definition keys by using:

```
assertThat(formDefinitions)
    .extracting(FormDefinition::getKey)
    .containsExactlyInAnyOrder("form1", "form2");
```

Reply:
Thanks; done.